backend/curriculum_tracking/api_views.py

```diff
@@ -862,6 +862,34 @@ def assign_user_as_reviewer(self, request, pk=None):
         todo
 
     # TODO: bulk set due dates
+    @action(
+        detail=False,
+        methods=["post", "get"],
+        serializer_class=serializers.SetBulkDueDateSerializer, # Doesn't have right fields.
+        permission_classes=[permissions.IsAdminUser],
+    )
+    def set_bulk_due_date(self, request, pk=None):
+        ''' WIP
+        Operations:
+        - Find Agile cards that are projects and have correct content_item_id.
+        - Filter for matching flavours, then set due time.
+        - Need to do this for each team.
+
+        Endpoint:
+        - Need team, content_item_id, flavours, due_time
+        - Serializer with requered fields?
+        '''
+        def get_cards():
+            for card in (
+                models.AgileCard.objects.filter(assignees__active_in=[True])
+                .filter(content_item__content_type=ContentItem.PROJECT)
```

Comment:
Good start.

You dont need to filter by content_item__content_type. Rather content_item_id. We wont be giving every single project the same due date, but we might be giving every single "Person" card the same due date.

Then it's important to filter by team as well. We might want to say that everyone in a specific bootcamp should have the same due date for `Katas level 0`